playground/src/hooks/useConfig.ts

```diff
@@ -0,0 +1,39 @@
+import { Config } from '@pandacss/types'
+import { useEffect, useState } from 'react'
+import * as pandaDefs from '@pandacss/dev'
+
+export const useConfig = (_config: string) => {
```

Comment:
might be out of scope for this PR but what do you think of supporting TS in the config ? currently it silently won't work, I'm thinking we could use esbuild-wasm for that or sucrase or some lightweight way to transpile it before the eval

Reply:
Makes sense to me @astahmer. Can work on this in a different PR